_data/projects/powershell-docker.yml

```
@@ -0,0 +1,10 @@
+name: PowerShell Docker Images
+desc: The docker images PowerShell releases, both official and community.
+site: https://github.com/powershell/powershell-docker
+tags:
+  - powershell
+  - docker
+  - oss
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/powershell/powershell-docker/labels/up-for-grabs
```

@TravisEz13 Just crosschecked the label name on your repo and found out that it is 'Up-for-Grabs' and not 'up-for-grabs'. Hence, the reason for the build failures.
```suggestion
  link: https://github.com/PowerShell/PowerShell-Docker/labels/Up-for-Grabs
```